main.js

```diff
@@ -26,136 +26,99 @@ categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
 
 //Add functions below
+function show(element) {
+  element.classList.remove('hidden');
+}
+
+function hide(element) {
+  element.classList.add('hidden');
+}
+
 function submitForm() {
```

Comment:
Need to pass in "event" as a parameter when using event.preventDefault.

Reply:
I think I see what you're getting at but I don't think it applies in this case. Here `element` is not an event; it's a placeholder parameter that just means "html element".

Reply:
Ohhhhh nevermind, this GH interface confused me and I thought you were talking about line 33, not line 37. Got it, and agreed. I'll change and push now/soon.